cmd/cloudshell_open/deploy.go

```diff
@@ -133,13 +145,16 @@ func generateRevisionName(name string, objectGeneration int64) string {
 }
 
 // patchService modifies an existing Service with requested changes.
-func patchService(svc *runapi.Service, envs map[string]string, image string) *runapi.Service {
+func patchService(svc *runapi.Service, envs map[string]string, image string, options options) *runapi.Service {
 	// merge env vars
 	svc.Spec.Template.Spec.Containers[0].Env = mergeEnvs(svc.Spec.Template.Spec.Containers[0].Env, envs)
 
 	// update container image
 	svc.Spec.Template.Spec.Containers[0].Image = image
 
+	// update the resources
+	svc.Spec.Template.Spec.Containers[0].Resources = optionsToResourceRequirements(options)
```

Comment:
🤔 should we be patching existing services with defaults?

imagine I deployed a previous revision with default (512mib), then I go  and update manually to 1GiB, then redeploy using button, now I get back 512Mib.

so I think we should omit this part of the change.

Reply:
Good point! Yeah, we should not.